services/QuillLMS/engines/evidence/app/services/evidence/gen_ai/evaluation/verbose/checker.rb

```diff
@@ -8,16 +8,16 @@ class Checker < Evaluation::BaseChecker
           MAX_SIZE = 325
           MAX_SENTENCES = 4
 
-          def run
-            return suboptimal if too_long?
-
-            optimal
-          end
+          VERBOSE = 'verbose'
+          SENTENCE_ERROR = "#{VERBOSE}.five_sentences"
+          LENGTH_ERROR = "#{VERBOSE}.325_characters"
```

Comment:
Should the error message be coupled to the constant: "#{VERBOSE}.#{MAX_SIZE}_characters"